components/Molecule/fourColumnGrid.html

```diff
@@ -0,0 +1,65 @@
+<div class="component" title="fourColumnGrid" version="1.1">
+  <style>
+    .fourColumnGrid {
+      display: grid;
+      margin: 0 auto;
+      grid-template-columns: 1fr 1fr 1fr 1fr;
+    }
+
+    .fourColumnGrid img {
+      width: 100%;
+      margin-bottom: 1em;
+    }
+
+    .fourColumnGrid figcaption {
+      text-align: center;
+      display: grid;
+    }
+
+    .fourColumnGrid figcaption a {
+      text-decoration: none;
+    }
+  </style>
+
+  <markup>
+    <div class="fourColumnGrid">
+      <figure class="col-1">
+        <img src="https://via.placeholder.com/150">
+        <figcaption>
+          <span>Name</span>
+          <a href="#">Website: example.com</a>
+          <a href="#">GitHub Profile</a>
+          <a href="#">LinkedIn Profile</a>
+        </figcaption>
+      </figure>
```

Comment:
Using `figure` and `figcaption` here is not generic enough. It is specific to the Graduates page. It's best to build the component with something more generic, like this:

```
<div>
    <img src="https://via.placeholder.com/150">
</div>
```

Reply:
I tried to fix all the feedback if I did in the right way.